server/src/main/java/com/google/sps/utils/Ad.java

```diff
@@ -0,0 +1,225 @@
+/* 
+ * Author: Kira Toal
+ * Date: June 24, 2020
+ */
+package com.google.sps.utils;
+import java.util.List; 
+ 
+public class Ad {
+  // fields
+  public String id; 
```

Comment:
Make these private so you control access to them through the getters and setters.

Reply:
Got it